homeassistant/components/laundrify/strings.json

```diff
@@ -1,4 +1,19 @@
 {
+  "entity": {
+    "binary_sensor": {
+      "wash_cycle": {
+        "name": "Cycle"
+      }
+    },
+    "sensor": {
+      "power_sensor": {
+        "name": "[%key:component::sensor::entity_component::power::name%]"
+      },
+      "energy_sensor": {
+        "name": "[%key:component::sensor::entity_component::energy::name%]"
+      }
```

Comment:
If we reference the name provided by the device class anyways, we can just as well drop the translation key and let the entity fall back to the device class translation automatically.

Reply:
Addressed in https://github.com/home-assistant/core/pull/121378/commits/981301970f6188f44495544107e322d4ccba0875